Bump openfisca-us and show poverty change instead of absolute rates

MaxGhenis opened the pull request:
Bumps openfisca-us to incorporate minor CTC and EITC bug fixes, and pins the version for reproducibility. These don't noticeably change the charts, and since they occur before the program's eligibility threshold, they don't change any numbers in the report.

I also changed the poverty chart from showing the absolute child poverty rate to showing the change to the overall poverty rate.
<img width="1265" alt="image" src="https://user-images.githubusercontent.com/6076111/171794453-f10fd7f7-fdc2-4639-8e25-b7f72e98a0cf.png">

rickecon commented:
@MaxGhenis. I just went through the notebook and everything works (except for weird `kaleido` package errors when creating the images). I would say just do the following things and this PR is ready to be merged.
- [x] Add a vertical "Cambridge cash" label to the vertical dotted line in both `child-allowance.html` and `child-allowance.png`.
- [x] Add updated `.png` files that correspond to the new `baseline.html` and `cambridge-cash.html`.
- [x] Update the text of the README.md to say something like the following:
> This is the repository for code used in the CGO *Research in Focus*, "How does targeted cash assistance affect incentives to work?" by Max Ghenis (@MaxGhenis). This repository contains the following items:
* [`analysis.ipynb`](https://github.com/TheCGO/cambridge-cash-assistance/blob/main/analysis.ipynb) Jupyter notebook. An executable notebook you can use to replicate all the analysis in the article and creation of output and figures. You can either clone or download this repository and run the `analysis.ipynb` notebook on your machine locally. Or you can open it as a Google Colab notebook (https://colab.research.google.com/github/TheCGO/cambridge-cash-assistance/blob/main/analysis.ipynb) and run the notebook from your browser in which all the software and computation is hosted in the cloud.
* [`/output/`](https://github.com/TheCGO/cambridge-cash-assistance/tree/main/output) directory. This folder contains the `.html` files for the dynamic visualizations created in the notebook for these analyses and the corresponding static `.png` image files.

MaxGhenis commented:
Thanks @rickecon, I've addressed your comments. The png files for the first two charts didn't change, suggesting that either the openfisca-us updates don't affect this household (maybe the html stores a timestamp or something) or that the changes are too small to be detected for drawing the png.

rickecon commented:
@MaxGhenis. Thanks for the updates. Merging.
